namespaces/live.cloud-platform.service.justice.gov.uk/send-legal-mail-to-prisons-dev/resources/elasticache-staff-ui.tf

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# Send Legal Mail to Prisons Staff UI Elasticache
+################################################################################
+
+module "slmtp_staff_ui_elasticache_redis" {
+  source                  = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=7.0.0"
+  vpc_name                = var.vpc_name
+  application             = var.application
+  environment_name        = var.environment
+  is_production           = var.is_production
+  infrastructure_support  = var.infrastructure_support
+  team_name               = var.team_name
+  business_unit           = var.business_unit
+  number_cache_clusters   = var.number_cache_clusters
+  node_type               = "cache.t4g.micro"
+  engine_version          = "7.0"
+  parameter_group_name    = "default.redis7"
+  namespace               = var.namespace
+
```

Comment:
Is the auth_token_rotated_date parameter needed here?

Reply:
No, it is only required when u want to rotate/regenerate the auth_token